tortoise/tortoise.go

```diff
@@ -103,11 +101,11 @@ func (t *turtle) evict(ctx context.Context) {
 			ballotsNumber.Dec()
 			delete(t.ballotRefs, ballot.id)
 		}
-		for range t.layers[lid].blocks {
+		for range t.layer(lid).blocks {
 			blocksNumber.Dec()
 		}
```

Comment:
blocksNumber and atxsNumber can do Sub() to avoid looping?
i guess realistically there is just 1 layer each time in steady state.

Reply:
thanks changed